.github/workflows/platformDeploy.yml

```diff
@@ -379,18 +339,14 @@ jobs:
             exit 1
           fi
 
-      - name: Upload web build to GitHub artifacts
-        uses: actions/upload-artifact@v4
-        with:
-          name: web-build
-          path: dist
+      - name: Upload web sourcemaps to GitHub Release
+        run: gh release upload ${{ github.event.release.tag_name }} dist/merged-source-map.js.map#web-sourcemap-${{ github.event.release.tag_name }} --clobber
```

Comment:
@roryabraham We should always upload web on staging and production, right? Then use the `--clobber` flag to update the file?

Reply:
this works, but it just means that when a staging version becomes a production version, the artifacts for the staging version with the same tag are deleted. I think that's fine though